_topics/en/low-r-grinding.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Low-R grinding
```

Comment:
I would probably use lowercase *r* for the signature component. It seems more common to use uppercase *R* to refer to the random point generated from the nonce, of which *r* is the x-coordinate.